google-cloud-pubsub/OVERVIEW.md

```diff
@@ -205,13 +205,16 @@ sleep
 Messages also can be pulled directly in a one-time operation. (See
 {Google::Cloud::PubSub::Subscription#pull Subscription#pull})
 
+The `immediate: false` option is now recommended to avoid adverse impacts on
```

Comment:
I would say "The `immediate: false` option is recommended to avoid adverse impacts on performance of pull operations." That way we are just a little more specific about the adverse impact> Also took out "now" as I think the natural question would be "when did 'now' start?" and so seems like we can just avoid that. If they do get interested in when the docs were updated, they can look at blame and see when it changed.